compliance/b_10.rst

```diff
@@ -0,0 +1,241 @@
+.. _compliance_b_10:
+
+====================================================
+170.315 (b)(10) Electronic Health Information Export
+====================================================
+
+`170.315 (b)(10) requires the implementation of Electronic Health Information (EHI) export functionality <https://www.healthit.gov/test-method/electronic-health-information-export>`_. 
+This encompasses the ability to generate export files for individual patients and entire patient populations in a computable format. 
+Firely Server provides comprehensive support for this requirement through its :ref:`Bulk Data Export feature <feature_bulkdataexport>`.
+For a deeper understanding and comprehensive utilization of this feature beyond the (b)(10) requirements, 
+please consult the provided documentation, which includes setup instructions, application settings, parameters, filtering options, and additional details.
+
+File Format
+===========
+
+Bulk Data Export files are formatted in NDJSON (New line delimited JSON), a modified version of the JSON format designed for efficient bulk data transfer. 
+NDJSON follows a simplified JSON structure where resources are serialized without whitespace and separated by a newline pair (characters 13 and 10). 
```

Comment:
To me it is not clear what you mean with "(characters 13 and 10)".  Maybe adding e.g. ASCII characters here can already help